microgateway_decorator/bin/compile

```diff
@@ -98,8 +98,32 @@ fi
 
 export PATH=$PATH:${BUILD_DIR}/tmp/node/bin
 # npm --prefix $BUILD_DIR/tmp/edgemicro install edgemicro
-git clone https://github.com/apigee-internal/microgateway.git $BUILD_DIR/tmp/edgemicro
-(cd $BUILD_DIR/tmp/edgemicro && git checkout tags/v2.5.8)
+
+# local installation if microgateway folder exists in lib directory
+if [ -d "$ROOT_DIR/lib/microgateway" ]; then
+  echo "Installing Edge Microgateway from lib directory."
+  cp -a $ROOT_DIR/lib/microgateway/. $BUILD_DIR/tmp/edgemicro
```

Comment:
ROOD_DIR/lib is the stuff inside the buildpack no the user code. User code is in BUILD_DIR

Reply:
Yeah that's an option, just wanted to make sure that you weren't intending on them putting the microgateway in the same directory as the code they push (kind of like how we do the custom plugins currently).